vars/odfRunDeployScript.groovy

```diff
@@ -0,0 +1,19 @@
+def call(){
```

Comment:
@aditidukle Since these are all using shell commands you can create shell script under dir /scripts/odf and directly call them from Jenkins file

Reply:
@aditidukle This comment is not addressed